nifi-assembly/pom.xml

```diff
@@ -192,6 +192,12 @@ language governing permissions and limitations under the License. -->
             <artifactId>nifi-hadoop-nar</artifactId>
             <version>1.12.0-SNAPSHOT</version>
             <type>nar</type>
+    	</dependency>
+	<dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-hdfs-repository-nar</artifactId>
```

Comment:
This cannot be in the convenience binary at this point.  We cannot afford to add nars unless their size is trivial (we're already over capacity in terms of build size).  Instead there should be instructions for users that want to experiment with this on how to add it to their nifi installs.

Reply:
since this nar depends on the hadoop nar it might be fine/small enough to include.  But we'd have to be clear this is experimental.

Reply:
Fair enough - should I put it in a different profile, or just completely remove it?

Reply:
Sorry comments overlapped. It doesn't depend on the hadoop-libraries-nar because it has to depend on the nifi-framework-nar in order for it to work properly with the NAR ClassLoader.

Reply:
The NAR is pretty big compared to most others: 53 MB where the average size is 23 MB. I'll add this to a new separate profile